source/manuals/logit-incident-management.html.md.erb

```diff
@@ -23,9 +23,9 @@ If you discover Logit or Kibana have failed, the process you must follow will di
 ### During office hours (9.30am to 5.30pm)
 
 1. Check the [Logit status page][] to see if Logit know about the incident.
-2. Join the #reliability-eng Slack channel and describe the problem.
+2. Join the [#reliability-eng Slack channel][] channel and describe the problem.
```

Comment:
this will render as:

> Join the [#reliability-eng Slack channel][] channel 

you have one "channel" too many

[#reliability-eng Slack channel]: https://govuk.slack.com